### PR TITLE
[8.0][I-215][BR1030]-Set Default Values

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
Issue #215 

- Set default Values for BR,DL,RL

### CODE REVIEW

File: business_requirement_deliverable/tests/test_br.py [hunk 1]
```diff
@@ -200,6 +200,7 @@ def test_resource_type_change(self):
         for line in self.br.deliverable_lines:
             for resource in line.resource_ids:
                 if resource and resource.resource_type == 'task':
+                    resource.resource_type_change()
```
Comment (elicoidal) [inline business_requirement_deliverable/tests/test_br.py]: Why triggering it twice?

Comment (YogeshMahera-SerpentCS) [reply business_requirement_deliverable/tests/test_br.py]: First method triggering for else condition then it write `resource_type=procurement` so second triggering for if condition.

Comment (martintorresvm) [reply business_requirement_deliverable/tests/test_br.py]: @YogeshMahera-SerpentCS Then please split into two test as is easier to understand.

Comment (YogeshMahera-SerpentCS) [reply business_requirement_deliverable/tests/test_br.py]: Ohky sure @victormartinelicocorp

Comment (YogeshMahera-SerpentCS) [reply business_requirement_deliverable/tests/test_br.py]: Ohky sure @victormartinelicocorp

**Review by elicoidal — APPROVED** [review]
Functional tests are OK
LGTM

**Comment by elicoidal** [comment]
@victormartinelicocorp

**Review by martintorresvm — APPROVED** [review]

**Pull request merged**